pkg/scclient/client.go

```diff
@@ -129,7 +130,9 @@ func (c *RegistryClient) Initialize(opt Options) (err error) {
 	}
 	//Update the API Base Path based on the Version
 	c.updateAPIPath()
-
+	c.pool = GetInstance()
```

Comment:
GetInstance() should accept address
delete setAddress, because it could be called many times

Reply:
> GetInstance() should accept address
> delete setAddress, because it could be called many times

can't understand.
my idea is use singleton pattern to get the instance
can change the address at runtime